packages/ui-components/src/chat/chat.tsx

```diff
@@ -99,6 +98,12 @@ export interface ChatProps extends InteractiveComponentProps<Message[]> {
     onUpdate?: (value: Message[]) => void | Promise<void>;
     /** The user who is currently active in the chat */
     activeUser: UserData;
+    /** The title to appear at the top of the chat */
+    chatTitle?: string;
+    /** The placeholder text for the reply field */
+    placeholder?: string;
+    /** Whether the chat is in a popup and should be styled as such */
+    isPopup?: boolean;
```

Comment:
Would it be more composable instead if you exposed effectively this component without the wrapper (so 'ChatBody', ReplyWrapper + the logic, intended to be rendered directly) as a 'Chat' component and then a 'ChatPopup' which would wrap the first component? Or are the popup visuals partially in dara-components? 
Feels like it could be a nicer pattern than having to do conditional styling, this way the place where we use it can choose to use its own wrapper if need be rather than be stuck with the two provided options. 

If it's urgent I'm fine with the current approach but I think it could maybe be improved